climada/test/test_plot.py

```diff
@@ -25,16 +25,28 @@
 import contextily as ctx
 import urllib
 
+from rasterio.windows import Window
 from climada.entity.entity_def import Entity
 from climada.hazard.base import Hazard
 from climada.entity.exposures.base import Exposures
 from climada.entity.impact_funcs.impact_func_set import ImpactFuncSet
 from climada.engine import ImpactCalc, ImpactFreqCurve
-from climada.util.constants import HAZ_DEMO_MAT, ENT_DEMO_TODAY
+from climada.util.constants import HAZ_DEMO_FL, HAZ_DEMO_MAT, ENT_DEMO_TODAY
+
+# Define an exposure to use for plotting
+exp = Exposures.from_raster(HAZ_DEMO_FL, window= Window(10, 20, 50, 60))
+exp.set_geometry_points()
```

Comment:
Is there any particular reason that you use `set_geometry_points`?

Reply:
Not anymore. At first I created an exposure from the method good_exposures() of /Exposure/test.base.py and there it was needed since no geometry column was created by the method good_exposures, I had to create it from lat and lon with set_geometry_points. By using the method from_raster you are right, set_geometry_points is no longer needed.